### PR TITLE
Add Google Play And F-Droid Badges

### PR DESCRIPTION
**Pull request by HiramLevitch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ Memento
 
 **Download** https://github.com/yaa110/Memento/releases
 
+[![Memento on Google Play](https://cloud.githubusercontent.com/assets/23243656/19977341/8d6d4be6-a1f3-11e6-9f10-bf53a4e43bf1.png)](https://play.google.com/store/apps/details?id=github.yaa110.memento)
+[![Memento on Google Play](https://user-images.githubusercontent.com/32419723/31298395-444a0bce-aaea-11e7-8624-cd38636fbf42.png)](https://f-droid.org/packages/github.yaa110.memento/)
```
Comment (yaa110) [inline README.md]: It should be `Memento on F-Droid`

Comment (yaa110) [inline README.md]: It should be `Memento on F-Droid`

**Comment by yaa110** [comment]
Since I did not publish this app in Play Store, please add a comment to announce that the published app in play store is unofficial

**Comment by HiramLevitch** [comment]
@yaa110 Done 👌

Do you know if the uploader is trustworthy? Do you plan to upload the app to Google Play yourself?

**Comment by yaa110** [comment]
@HiramLevitch Please merge commits. I have no idea about the publisher, however, I can't publish it to play store.

**Comment by HiramLevitch** [comment]
OK.
Can you please click on ```Merge``` as I do not have write access to this repository. Thank you! ☺️

**Pull request merged**